deepinv/tests/test_physics.py

```diff
@@ -96,6 +96,9 @@ def find_operator(name, device):
         factor = 2
         norm = 1 / factor**2
         p = dinv.physics.Downsampling(img_size=img_size, factor=factor, device=device)
+    elif name == "random_phase_retrieval":
+        img_size = (1, 32, 32)
+        p = dinv.physics.RandomPhaseRetrieval(m=10, img_shape=img_size, device=device)
```

Comment:
Nice! But unless I'm mistaken this is not tested. To do so, you'd need to add `random_phase_retrieval` in here:
```
NONLINEAR_OPERATORS = ["haze", "blind_deblur", "lidar", "random_phase_retrieval"]
```

Reply:
Thanks for the guidance, currently I don't add `random_phase_retrieval` to the tests because it is different from other linear physics, and I am not sure if the tests for linear physics should also be applied to `random_phase_retrieval`. After today's meeting with Julian, I knew they are indeed needed and will work on it.